terraform/groups/configurable-api-caller/profiles/live-eu-west-2/dissolutions_submit.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Will this code remain unran until we explicitly initialise it? Just don't want a lambda to be running in production when the endpoint isn't available and spamming 404s

Reply:
Correct, nothing will be deployed on production until we explicitly do so. We can also target the required resources to only deploy what is needed in case we need to do partial deploys.

If we exclude this file, we would have a partial setup for the production deploy code as the main.tf file containing the resources is shared across all environments, but it would be missing dependancies.